docs/BuildFromSource.md

```diff
@@ -45,104 +45,109 @@ Visual Studio 2019 (16.8) is required to build the repo locally. If you don't ha
 
 > :bulb: Be sure you have least 10 GB of disk space and a good Internet connection. The build scripts will download several tools and dependencies onto your machine.
 
 #### curl/wget
 
 `curl` and `wget` are command line tools that can be used to download files from an HTTP server. Either utility will need to be installed in order to complete the setup. Typically, these will be included on your machine by default.
 
-If netier utility is installed, you can install curl (https://curl.haxx.se) or wget (https://www.gnu.org/software/wget).
+If neither utility is installed, you can install curl (<https://curl.haxx.se>) or wget (<https://www.gnu.org/software/wget>).
 
-##### Git
+#### Git
 
 If you've made it this far, you've already got `Git` installed. Sit back, relax, and move on to the next requirement.
 
 #### [NodeJS](https://nodejs.org)
 
-Building the repo requires version 10.14.2 or newer of Node. You can find installation executables for Node at https://nodejs.org.
+Building the repo requires version 10.14.2 or newer of Node. You can find installation executables for Node at <https://nodejs.org>.
 
 #### [Yarn](https://yarnpkg.com/)
 
-NodeJS installs the Node package manager (npm) by default. This repo depends on yarn, an alternate package manager for the Node ecosystem. You can install Yarn from the command line using the following command.
+NodeJS installs the Node package manager (npm) by default. This repo depends on Yarn, an alternate package manager for the Node ecosystem. You can install Yarn from the command line using the following command.
 
 ```bash
-$ npm install -g yarn
+npm install -g yarn
 ```
 
 #### Java Development Kit
 
 This repo contains some Java source code that depends on an install of the JDK v11 or newer. The JDK can be installed from either:
-  * OpenJDK <https://jdk.java.net/>
-  * Oracle's JDK <https://www.oracle.com/technetwork/java/javase/downloads/index.html>
+
+- OpenJDK <https://jdk.java.net/>
+- Oracle's JDK <https://www.oracle.com/technetwork/java/javase/downloads/index.html>
 
 Similar to [the instructions above for Windows](#java-development-kit-in-windows), be sure that the the `JAVA_HOME` environment variable is set to the location of your Java installation.
 
 ## Step 3: Build the repo
 
-Before opening our .sln/.slnf files in Visual Studio or VS Code, you will need to build the repo locally.
+Before opening our .sln/.slnf files in Visual Studio or VS Code, you will need to at least restore the repo locally.
 
 ### In Visual Studio
 
-To set up your project for development on Visual Studio, you'll need to execute the following command.
+To set up your project for development on Visual Studio, you'll need to execute the following command. Building
+subsets of the repo e.g. Java projects may (depending on your scenario) be necessary before building within Visual
+Studio because those projects are not listed in AspNetCore.sln.
```

Comment:
new

Reply:
> Studio because those projects are not listed in AspNetCore.sln.

Curious: why is this the case?

Reply:
Nope, no clue how to add javaproj or npmproj to VS.

Nit: "Npm projects" instead of "Java projects" above, no one depends on a Java project.